oss_src/sframe_query_engine/planning/planner.cpp

```diff
@@ -52,7 +51,7 @@ static sframe execute_node_impl(pnode_ptr input_n, const materialize_options& ex
  * Executes a query plan, potentially parallelizing it if possible.
  * Also implements fast paths in the event the input node is a source node.
  */
-static sframe execute_node(pnode_ptr input_n, const materialize_options& exec_params) {
+static sframe execute_node(pnode_ptr input_n, materialize_options exec_params) {
```

Comment:
good catch. It was modified when I was trying to change the execution component and I forget to change it back. Will fix.